plugins/modules/proxmox_backup_info.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024 Marzieh Raoufnezhad <raoufnezhad at gmail.com>
+# Copyright (c) 2024 Maryam Mayabi <mayabi.ahm at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: proxmox_backup_info
+
+short_description: Retrieve information on Proxmox scheduled backups
+
+description:
+  - Retrieve information such as backup times, VM name, VM ID, mode, backup type, and backup schedule using the Proxmox Server API.
+
+author:
+  - "Marzieh Raoufnezhad (@raoufnezhad) <raoufnezhad@gmail.com>"
+  - "Maryam Mayabi (@mmayabi) <mayabi.ahm@gmail.com>"
+
+options:
+  vm_name:
+    description: The name of the Proxmox VM.
+    required: false
+    type: str
+  vm_id:
+    description: The ID of the Proxmox VM.
+    required: false
+    type: int
+  backup_section:
+    description: The backup_section value is true or false. if this be true, this module just return all backup job information.
+    required: false
+    default: false
+    type: bool
+
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+  - community.general.proxmox.actiongroup_proxmox
+"""
+
+EXAMPLES = """
+- name: Print all backup information by VM id and VM name
+  proxmox_backup_info:
+      api_user: 'myUser@pam'
+      api_password: '*******'
+      api_host: '192.168.20.20'
```

Comment:
Please use an indentation of 2.
```suggestion
    api_user: 'myUser@pam'
    api_password: '*******'
    api_host: '192.168.20.20'
```

(Same for the other examples.)